### DISCUSSION TITLE
Will it be possible to sf project deploy start to deploy without source tracking

### DISCUSSION THREAD

**Discussion by amarek-cytiva** [forcedotcom/cli#2497] [General]
Hi All,
I was trying to migrate from the previous sfdx commands to the new sf command but I've faced some serious problems. 

I cannot run the 
`sf project deploy start `
on the organization that has source tracking enabled because of conflicts.
Of course, it is possible to run sf project deploy validate but this command does not have an --ignore-warning flag.

The command that is working correctly is 
`sf force:source:deploy `
but this command already has a warning that will be deprecated.
I do not need to be worried about the conflict which I do not care about at that time as the code base that I'm trying to deploy is the one that I want to have in the sandbox.

I suspected that the command alignment would not change the overall behavior but now it is. 

I would like to know your opinion on this and maybe someone from the creator could put some more light on this one.

**Comment by mshanemc** [comment]
If a scratch org, use `sf org create scratch --no-track-source` if you know you don't need tracking.  But it sounds like this is a sandbox.

And if you just want to override all the conflicts with your deploy, you can `sf project deploy start --ignore-conflicts`, right?

**Reply by amarek-cytiva** [reply]
Thank you for the answer, that was it,
Yes, we are deploying the code to sandboxes as we do not see the benefits of using scratch orgs.
I don't know did I missed this flag before but that is exactly what was needed in order to start the process of validation.

**Comment by amarek-cytiva** [comment]
But now there is another problem,
as in our CI process, we are reading the status asynchronously and I'm using --async flag.
However I do not see --target-org or any other flag which could be use to read the deployment status.

If I won't provide the user flag then I will get the error that job was not found

**Reply by mshanemc** [reply]
oh, then you want yesterday's release.  

https://github.com/forcedotcom/cli/blob/main/releasenotes/README.md#2118-oct-4-2023-stable

**Reply by amarek-cytiva** [reply]
wow :) that was quick. Working perfectly

**Comment by mshanemc** [comment]
Note that if you have an org you'd like to disable tracking for, you can do that with the newish `org enable|disable tracking` commands

https://github.com/forcedotcom/cli/blob/main/releasenotes/README.md#2197-nov-29-2023
